charts/kubewarden-controller/templates/webhooks.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "kubewarden-controller.fullname" . }}-serving-cert
   name: kubewarden-controller-mutating-webhook-configuration
+  namespace: {{ .Release.Namespace }}
```

Comment:
we don't need this here since `MutatingWebhookConfiguration` is not a namespaced resource.